### PR TITLE
feat: support for deciding category on basis of pull request title 

### PR DESCRIPTION
**Pull request by ShrutiVerma97**
Following label types added:
 ['feat','fix', 'docs', 'style', 'refactor', 'perf', 'test', 'build', 'ci', 'chore', 'revert']

### CODE REVIEW

**Comment by krishna-kb** [comment]
Fix tests.

File: lib/releases.js [hunk 1]
```diff
@@ -5,6 +5,19 @@ const { template } = require('./template')
...
+  'docs',
+  'style',
+  'refactor',
+  'perf',
+  'test',
+  'build',
+  'chore',
+  'revert'
+]
+const regexFromMyArray = new RegExp(types.join('|'), 'gi')
```
Comment (vishwanathd11) [inline lib/releases.js:20]: regexFromMyArray could use better name.

**Comment by vishwanathd11** [comment]
@ShrutiVerma97 use commitizen for doing commits or else if you are doing them manually follow the commit message guidelines. Lets add commit hooks of those guidelines here as well. You may copy from mobile-web-dream11.

File: test/index.test.js [hunk 2]
```diff
@@ -539,260 +543,13 @@ Previous tag: ''
...
-                body: `# What's Changed
-* Update dependencies (#4) @TimonVS
-## 🚀 Features
 * Add big feature (#2) @TimonVS
 * 👽 Add alien technology (#1) @TimonVS
-## 🐛 Bug Fixes
-* Bug fixes (#3) @TimonVS
-`,
-                draft: true,
-                tag_name: ''
```
Comment (vishwanathd11) [inline test/index.test.js:795]: I am assuming we deleted not working test case to make test cases pass. We will also have to update/add the test cases as per the changes done by us.

Comment (ShrutiVerma97) [reply test/index.test.js:795]: @vishwanathd11 Yes, I've removed those test cases for labels, I am writing new ones now as per our needs.

**Comment by ShrutiVerma97** [comment]
> @ShrutiVerma97 use commitizen for doing commits or else if you are doing them manually follow the commit message guidelines. Lets add commit hooks of those guidelines here as well. You may copy from mobile-web-dream11.

okay

File: README.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       # Drafts your next Release notes as Pull Requests are merged into "master"
-      - uses: release-drafter/release-drafter@v5
+      - uses: dream11/release-drafter@v6
```
Comment (vishwanathd11) [inline README.md:27]: You may add a section at the top of the README file telling from which open source repo we have forked this. You may mention commit as well. In future we might want to take update, although probability of that will be very less.

Comment (vishwanathd11) [reply README.md:27]: This can be done in a separate PR.

**Comment by vishwanathd11** [comment]
Please follow the semantic commit message guidelines going forward.

**Pull request merged**